CMakeLists.txt

```diff
@@ -1,40 +1,43 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.11)
 project(ostis-metasystem)
 set(CMAKE_CXX_STANDARD 17)
 
-option(SC_BUILD_TESTS "Flag to build unit tests" OFF)
+option(SC_BUILD_TESTS "Flag to build unit tests" ON)
```

Comment:
Should it be ON by default? I don't think so, since our installation strategy is still build-based and a lot of users will build test files but will never use them